src/main/java/uk/gov/hmcts/reform/pip/account/management/controllers/AccountController.java

```diff
@@ -337,4 +337,15 @@ public ResponseEntity<? extends PiUser> createSystemAdminAccount(
         return ResponseEntity.ok(systemAdminAccountService.addSystemAdminAccount(account, issuerId));
     }
 
+    @ApiResponses({
+        @ApiResponse(responseCode = OK_CODE, description = "System Admin Information"),
```

Comment:
Response here should be updated if making generic

Reply:
fixed.